index.html

```diff
@@ -154,7 +154,48 @@ <h3 class="skills-head"><img class="skill-bullets" src="./Work-section-img/squar
           </div>
         </div>
         </div>
+        </div>
+         
+        <div class="contact">
+            <div class="contact-grid">
+            <div class="contactpara">
+                <p class="contactp">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <div class="form">
+                <form class="form1" action="https://formspree.io/f/mgebknld" method="post">
+                    <div class="name">
+                    <input type="text" name="username" placeholder="Fullname" class="name1" maxlength="30" required>
+                    <div class="blank1"></div>
+                    </div>
+                    <div class="email">
+                    <input type="email" name="Email-address" placeholder="email address" class="email1" required>
+                    <div class="blank2"></div>
+                    </div>
+                    <div class="text">
+                    <textarea name="comment" placeholder="Enter text here" class="text1" maxlength="500" required></textarea>
+                    </div>
+                    <button name="submit" class="submit">Get in touch</button>
+                    </form>
```

Comment:
- [ ] Kindly let's stick to the Figma design as much as possible for the Form also 😀.

- Your design
<img width="137" alt="wrong" src="https://user-images.githubusercontent.com/41163571/220544182-140ea6a7-adbd-4012-94e3-8d7720f26bb4.png">

- Figma design
<img width="269" alt="right" src="https://user-images.githubusercontent.com/41163571/220544256-90aed3a2-e6d4-4a11-a92d-a3d8762b18da.png">

😀. By giving padding to the input elements more, ensuring the placeholders are the exact ones in the Figma design in terms of the spacing, the casing(upper or lower case and the likes), and likewise removing the `border` or `outline` properties of the input elements but on `focus` and off `focus`, please check [this](https://www.w3schools.com/cssref/sel_focus.php) for more info. 😉